pre_commit/commands/run.py

```diff
@@ -259,6 +259,11 @@ def run(config_file, store, args, environ=os.environ):
         environ['PRE_COMMIT_ORIGIN'] = args.origin
         environ['PRE_COMMIT_SOURCE'] = args.source
 
+    # Expose remote name and url as environment variables for hooks to consume
+    if args.push_remote_name and args.push_remote_url:
+        environ['PRE_PUSH_REMOTE_NAME'] = args.push_remote_name
+        environ['PRE_PUSH_REMOTE_URL'] = args.push_remote_url
```

Comment:
these should be `PRE_COMMIT_...` since they come from the the tool named `pre-commit` (as per above)

Reply:
You mean PRE_PUSH_REMOTE_NAME should be PRE_COMMIT_REMOTE_NAME and the same with remote url?

Reply:
yes